modules/administration-guide/partials/proc_configuring-che-with-custom-resources-to-use-amazon-s3.adoc

```diff
@@ -0,0 +1,62 @@
+[id="configuring-{prod-id-short}-with-custom-resources-to-use-amazon-s3_{context}"]
+= Configuring {prod-short} with custom resources to use Amazon S3
+
+To configure {prod-short} to use Amazon S3:
+
+.Prerequisites
+
+* Configured Amazon S3. See xref:supported-restic-compatible-backup-servers.adoc[].
+
+.Procedure
+
+. Create a secret containing the `repo-password` key with a Base64 encoded password:
++
+[source,yaml,subs="+quotes,+attributes"]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: backup-encryption-password-secret
+  namespace: eclipse-che
+type: Opaque
+stringData:
+  repo-password: __<base64_encoded_password>__
+----
++
+({prod-short} Operator will set up this password for the backup repository that {prod-short} Operator will create from this custom object on the backup server.)
```

Comment:
```suggestion
({prod-short} Operator sets up this password for the backup repository that {prod-short} Operator creates from this custom object on the backup server.)
```
Avoid future tense

Reply:
@bburt-rh, thank you for spotting it, but in this particular case I considered future tense warranted. IBM Style: "_Use past or future tense only when you cannot use present tense or it does not make sense to use present tense._"